sdk/src/types/block/output/basic.rs

```diff
@@ -198,6 +273,27 @@ impl BasicOutputBuilder {
     }
 }
 
+impl Rent for BasicOutputBuilder {
```

Comment:
This is so tedious and error prone 😭

Reply:
yeah, you're not wrong. But what choice do we have?